### PR TITLE
Dev daniel

### PR DESCRIPTION
**Pull request by danielbattistini**
Update Lc analysis.

### CODE REVIEW

File: configfiles/fit/config_Dplus_Fit_pp5TeV.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 pp5TeVPrompt:
-    Meson: 'Dplus'
+    Particle: 'Dplus'
```
Comment (fgrosa) [inline configfiles/fit/config_Dplus_Fit_pp5TeV.yml:2]: This does not work unless you include also the modification to the https://github.com/DmesonAnalysers/DmesonAnalysis/blob/master/GetRawYieldsDplusDs.C and https://github.com/DmesonAnalysers/DmesonAnalysis/blob/master/GetRawYieldsDplusDs.py scripts

File: configfiles/fit/config_Dplus_Fit_pp5TeV.yml [hunk 2]
```diff
@@ -21,7 +21,7 @@ pp5TeVPrompt:
 
 # prompt in FD bins
 # pp5TeVPrompt:
-#     Meson: 'Dplus'
+#     Particle: 'Dplus'
```
Comment (fgrosa) [inline configfiles/fit/config_Dplus_Fit_pp5TeV.yml:24]: same comment as above

File: configfiles/fit/config_Dplus_Fit_pp5TeV.yml [hunk 3]
```diff
@@ -42,7 +42,7 @@ pp5TeVPrompt:
 #     SgnFunc: [ 'kGaus', 'kGaus', 'kGaus', 'kGaus', 'kGaus', 'kGaus', 'kGaus', 'kGaus' ]
 
 pp5TeVFD:
-    Meson: 'Dplus'
+    Particle: 'Dplus'
```
Comment (fgrosa) [inline configfiles/fit/config_Dplus_Fit_pp5TeV.yml:45]: same comment as above

File: configfiles/fit/config_Ds_Fit_PbPb.yml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 Cent010:
-    Meson: 'Ds'
+    Particle: 'Ds'
```
Comment (fgrosa) [inline configfiles/fit/config_Ds_Fit_PbPb.yml:2]: same comment as above

File: configfiles/fit/config_Ds_Fit_PbPb.yml [hunk 5]
```diff
@@ -20,7 +20,7 @@ Cent010:
     SgnFunc: [ 'kGaus', 'kGaus', 'kGaus', 'kGaus', 'kGaus', 'kGaus', 'kGaus', 'kGaus' ]
 
 Cent3050:
-    Meson: 'Ds'
+    Particle: 'Ds'
```
Comment (fgrosa) [inline configfiles/fit/config_Ds_Fit_PbPb.yml:23]: same comment as above

File: configfiles/fit/config_Ds_Fit_PbPb_ML.yml [hunk 6]
```diff
@@ -1,5 +1,5 @@
 Cent010:
-    Meson: 'Ds'
+    Particle: 'Ds'
```
Comment (fgrosa) [inline configfiles/fit/config_Ds_Fit_PbPb_ML.yml:2]: same comment as above

File: configfiles/fit/config_Ds_Fit_PbPb_ML.yml [hunk 7]
```diff
@@ -20,7 +20,7 @@ Cent010:
     SgnFunc: ['kGaus', 'kGaus', 'kGaus', 'kGaus', 'kGaus', 'kGaus', 'kGaus', 'kGaus', 'kGaus', 'kGaus']
 
 Cent3050:
-    Meson: 'Ds'
+    Particle: 'Ds'
```
Comment (fgrosa) [inline configfiles/fit/config_Ds_Fit_PbPb_ML.yml:23]: same comment as above

File: configfiles/fit/config_Ds_Fit_pp5TeV.yml [hunk 8]
```diff
@@ -1,5 +1,5 @@
 pp5TeVPrompt:
-    Meson: 'Ds'
+    Particle: 'Ds'
```
Comment (fgrosa) [inline configfiles/fit/config_Ds_Fit_pp5TeV.yml:2]: same comment as above

File: configfiles/fit/config_Ds_Fit_pp5TeV.yml [hunk 9]
```diff
@@ -20,7 +20,7 @@ pp5TeVPrompt:
     SgnFunc: ['kGaus', 'kGaus', 'kGaus', 'kGaus', 'kGaus', 'kGaus', 'kGaus', 'kGaus', 'kGaus']
 
 pp5TeVFD:
-    Meson: 'Ds'
+    Particle: 'Ds'
```
Comment (fgrosa) [inline configfiles/fit/config_Ds_Fit_pp5TeV.yml:23]: same comment as above

File: configfiles/fit/config_Ds_Fit_pp5TeVStd.yml [hunk 10]
```diff
@@ -1,5 +1,5 @@
 pp5TeVPrompt:
-    Meson: 'Ds'
+    Particle: 'Ds'
```
Comment (fgrosa) [inline configfiles/fit/config_Ds_Fit_pp5TeVStd.yml:2]: same comment as above

File: configfiles/fit/config_Lc_Fit_pp13TeV_highSignif.yml [hunk 11]
```diff
@@ -0,0 +1,20 @@
+pp13TeVFD:
```
Comment (fgrosa) [inline configfiles/fit/config_Lc_Fit_pp13TeV_highSignif.yml:1]: To reduce a bit the number of config files, you could have a single config file for the fits with the two entries `pp13TeVFD` and `pp13TeVPrompt`

File: configfiles/fit/config_Lc_Fit_pp13TeV_highFD.yml [hunk 12]
```diff
@@ -0,0 +1,20 @@
+pp13TeVFD:
```
Comment (fgrosa) [inline configfiles/fit/config_Lc_Fit_pp13TeV_highFD.yml:1]: Also this does not work if you don't modify also the https://github.com/DmesonAnalysers/DmesonAnalysis/blob/master/GetRawYieldsDplusDs.C and https://github.com/DmesonAnalysers/DmesonAnalysis/blob/master/GetRawYieldsDplusDs.py scripts

File: configfiles/fit/config_Lc_Fit_pp13TeV_highFD.yml [hunk 13]
```diff
@@ -0,0 +1,20 @@
...
+    Particle: 'Lc'
```
Comment (fgrosa) [inline configfiles/fit/config_Lc_Fit_pp13TeV_highFD.yml:2]: same as above

**Review by fgrosa — CHANGES_REQUESTED** [review]
Ciao @danielbattistini thanks a lot! I have a couple of comments, but the most important one is that the config files that you added / modified are not comaptible with the scripts for the raw yields that are in the repository, so it would be better to commit also the changes on them.

File: configfiles/fit/config_Dplus_Fit_pp5TeV.yml (hunk 1 above)
Comment (danielbattistini) [reply configfiles/fit/config_Dplus_Fit_pp5TeV.yml:2]: I did it but I had some problems with git history, I didn't realize those files were not included. They will be updated in the next commit.

File: configfiles/fit/config_Lc_Fit_pp13TeV_highSignif.yml (hunk 11 above)
Comment (danielbattistini) [reply configfiles/fit/config_Lc_Fit_pp13TeV_highSignif.yml:1]: Thanks! I didn't think of this possibility

File: GetRawYieldsDplusDs.C [hunk 14]
```diff
@@ -42,8 +42,11 @@ void DivideCanvas(TCanvas* c, int nPtBins);
 
 //__________________________________________________________________________________________________________________
 int GetRawYieldsDplusDs(int cent, bool isMC, TString infilename, TString cfgfilename, TString outFileName) {
+    std::cout << __LINE__<<std::endl;
```
Comment (fgrosa) [inline GetRawYieldsDplusDs.C:45]: are these lines needed?

File: GetRawYieldsDplusDs.C [hunk 15]
```diff
@@ -274,7 +304,13 @@ int GetRawYieldsDplusDs(int cent, bool isMC, TString infilename, TString cfgfile
     //fit histos
     double massDplus = TDatabasePDG::Instance()->GetParticle(411)->Mass();
     double massDs = TDatabasePDG::Instance()->GetParticle(431)->Mass();
-    double massForFit = (meson==kDplus) ? massDplus : massDs;
+    double massLc = TDatabasePDG::Instance()->GetParticle(4122)->Mass();
+    double massForFit;
+    
+    if(particle==kDplus) massForFit = massDplus;
+    else if (particle==kDplus) massForFit = massDs;
```
Comment (fgrosa) [inline GetRawYieldsDplusDs.C:311]: here `Dplus` should be `Ds`

**Review by fgrosa — CHANGES_REQUESTED** [review]

**Review by fgrosa — APPROVED** [review]

**Pull request merged**